integration/index_test.go

```diff
@@ -80,8 +80,8 @@ func prepareSociIndices(t *testing.T, sh *dockershell.Shell) []testImageIndex {
 	return testImages
 }
 
-func getZtocDigests(sh *dockershell.Shell, img imageInfo) ([]string, error) {
-	ztocInfoBytes := sh.O("soci", "ztoc", "list")
+func getZtocDigestsForImage(sh *dockershell.Shell, img imageInfo) ([]string, error) {
+	ztocInfoBytes := sh.O("soci", "ztoc", "list", "--image-ref", img.ref)
 	scanner := bufio.NewScanner(bytes.NewReader(ztocInfoBytes))
```

Comment:
Maybe we should add a `quiet` flag to avoid this.

Reply:
Agreed. I suppose that can be a separate PR

Reply:
We could also add it to `soci index list` too

Reply:
We already have it. :) 

https://github.com/awslabs/soci-snapshotter/blob/bff4b07408aa3df86d5932a036880c5bf2ff6f4a/cmd/soci/commands/index/list.go#L73-L76

Reply:
Nice didnt catch that